experimental/packages/opentelemetry-api-metrics/src/internal/semver.ts

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { VERSION } from '../version';
```

Comment:
This seems like a duplication of `api/internal/semver.ts`. I'm wondering that currently the API trace/metrics are split into separate packages in order to reduce the package size after bundling (with tree shaking). This duplication would inevitably increase the bundle size if the end application uses both API packages.

Not an issue to release the first GA version of api-metrics. I'll dig into the tree-shaking to see what we can do to reduce the duplication.

Reply:
it is an exact copy. I thought about reusing the code but it is specifically under an internal namespace and has no stability guarantee. we could also create a new package but this was the easiest way to unblock the metrics release.

Reply:
I'd be cautious about introducing a new util package -- specifically, we've got `@opentelemetry/core`. As the core package is depending on the API package, I assume it is not possible for us to reuse it for API utils.

+1 on the internal namespace, it can be simpler IMO. Either way, I'm not suggesting doing it right now.